natural-language-understanding/v1.ts

```diff
@@ -14,653 +14,15 @@
  * limitations under the License.
  */
 
-import * as extend from 'extend';
-import { RequestResponse } from 'request';
-import { createRequest } from '../lib/requestwrapper';
-import { getMissingParams } from '../lib/helper';
-import { BaseService } from '../lib/base_service';
+import GeneratedNaturalLanguageUnderstandingV1 = require('./v1-generated');
```

Comment:
Can you use `import {...} from ....`?

Reply:
Yes ... right ... should have done that.  I will fix.

Reply:
Actually ... no ... can't do that.  It fails with
```
natural-language-understanding/v1.ts(17,10): error TS2305: Module '"/Users/mkistler/Projects/watson-node/wdc-node-sdk/natural-language-understanding/v1-generated"' has no exported member 'GeneratedNaturalLanguageUnderstandingV1'.
```

Reply:
At some point I think Ammar manually renamed the generated services to have that `Generated` prefix. Probably so he could use the `import { GeneratedService } from './v1-generated'` syntax while avoiding a naming conflict between the generated code and the adapter.

But the Node template generates `Service` instead of `GeneratedService`. So it looks like it got overwritten when more recent code was generated.

Do we have a strong preference one way or the other? I suppose users will be importing the `v1` adapter whether they use the legacy API or the new API—any calls with the new API will just be transparently passed along to the generated code. Since users don't directly import `v1-generated`, we could use whatever name we want there.

My preference, however, would be to continue generating `Service` instead of `GeneratedService` so that the generator is easier to use with external services (e.g. Watson Data Platform, Watson Finance, etc.)

Reply:
This also fails:
```
natural-language-understanding/v1.ts(18,8): error TS1192: Module '"/Users/mkistler/Projects/watson-node/wdc-node-sdk/natural-language-understanding/v1-generated"' has no default export.
```

Reply:
I think Glenn is right regarding the `Generated` name. I will merge this and we can fix this issue once we generate the service without `Generated`